.github/scripts/generate_release_notes.py

```diff
@@ -65,11 +66,17 @@ def get_repo_priority(name):
         return top_repos.index(name)
     return len(top_repos)
 
-# or using an access token
+# using an access token
 g = Github(githubToken)
 
 org = g.get_organization("dapr")
 
+# discover milestone project
+suffix = os.getenv("MILESTONE_REGEX_SUFFIX")
```

Comment:
now the project name for RC1 is `1.0.0 Milestone 1` 

This gives the flexibility for setting the suffix otherwise defaulting to Milestone.